tests/cache/test_redis_backend.py

```diff
@@ -57,6 +62,68 @@ class RedisDistributedMutexTest(_TestRedisConn, _GenericMutexTest):
     }
 
 
+class RedisAsyncCreationTest(_TestRedisConn, _GenericBackendFixture, TestCase):
+    backend = "dogpile.cache.redis"
+    config_args = {
+        "arguments": {
+            "host": REDIS_HOST,
+            "port": REDIS_PORT,
+            "db": 0,
+            "distributed_lock": True,
+            # This is the important bit:
+            "thread_local_lock": False,
```

Comment:
Commenting this out or setting it to True will show the negative case - the mutex.release() at line 94 will fail, the event will never be set at line 98, and the checks at the end of this test will fail.

Reply:
nice